cosmo_tester/framework/testenv.py

```diff
@@ -223,6 +224,9 @@ def __init__(self):
             tempfile.mkdtemp(prefix='cloudify-testenv-')
             for _ in xrange(self._additional_managers)]
 
+        if self._additional_managers:
```

Comment:
Not really.

On Thu, Oct 1, 2015 at 1:01 PM, konradxyz <notifications@github.com> wrote:

> In cosmo_tester/framework/testenv.py
> <https://github.com/codilime/cloudify-system-tests/pull/1#discussion_r40901400>
> :
>
> > @@ -223,6 +224,9 @@ def __init__(self):
> >              tempfile.mkdtemp(prefix='cloudify-testenv-')
> >              for _ in xrange(self._additional_managers)]
> >
> > +        if self._additional_managers:
>
> Is this if necessary?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codilime/cloudify-system-tests/pull/1/files#r40901400>
> .
>